docs/code_search/reference/queries.mdx

```diff
@@ -11,32 +11,52 @@ This page describes search pattern syntax and keywords available for code search
 
-### Standard search (default)
+### Keyword search (default)
 
-Standard search matches literal patterns exactly, including puncutation like quotes. Specify regular expressions inside `/./code_search/reference`.
+<Callout type="note" title="Experimental">New in version 5.3: Keyword search is in the experimental stage and enabled by default. Site administrators can disable keyword search globally by setting `experimentalFeatures.keywordSearch: false` in site settings. Users can override the setting in their user settings.</Callout>
+
+Keyword search matches individual terms anywhere in the document or the filename. Specify regular expressions inside `/.../`.
+
+|                                                              Search pattern syntax                                                               |                                                              Description                                                               |
+| ------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| [`foo bar`](https://sourcegraph.com/search?q=foo+bar&patternType=keyword)                                                                       | Match documents containing both `foo` and `bar` anywhere in the document. |
+| [`"foo bar"`](https://sourcegraph.com/search?q=%22foo+bar%22&patternType=keyword)                                                               | Match the string `foo bar` exactly. The space between the terms is interpreted literally. The quotes are not matched.                                                                |
+| [`"foo \"bar\""`](https://sourcegraph.com/search?q=context:global+%22foo+%5C%22bar%5C%22%22&patternType=keyword)                                                               | Match the string `foo "bar"` exactly.                                                                |
+| [`/foo.*bar/`](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+/foo.*bar/&patternType=keyword) | Match the **regular expression** `foo.*bar`.                      We support [RE2 syntax](https://golang.org/s/re2syntax).             |
+| [`foo OR bar`](https://sourcegraph.com/search?q=context:global+foo+OR+bar&patternType=keyword)                                            | Match documents containing `foo` _or_ `bar` anywhere in the document.                                                            |
+
+Matching is case-insensitive (toggle the `Aa` button to change).
+
+### Standard search
+
+<Callout type="note" title="Experimental">
```

Comment:
It may be confusing to have an experimental (or beta) badge next to Standard search. Maybe we can remove this badge and just include the info about the toggle in the Keyword search callout above?